lib/make-badge.js

```diff
@@ -89,7 +89,7 @@ function compressSvg(string, callback) {
   svgo.optimize(string, callback);
 }
 
-const cssColor = /^#([0-9a-fA-F]{6}|[0-9a-fA-F]{3})$/;
+const cssColor = /^#([0-9a-f]{3}){1,2}$/i;
```

Comment:
Could you leave a comment in the code that this is an optimized regex, with a link to this PR? Wouldn't want anyone to tweak it and undo this work!

Reply:
Good idea, will add that.